### PR TITLE
Fix python empty package bug + new training data.

### PR DESCRIPTION
**Pull request by NoahHakansson**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -10,8 +10,10 @@ import (
 )
 
 var (
-	pythonPath = os.Getenv("HOME") + "/.local/share/gosplat/src/python_helper/fast_model_compare.py"
-	modelBin   = os.Getenv("HOME") + "/.local/share/gosplat/fast-fb-model.bin"
+	// pythonPath = os.Getenv("HOME") + "/.local/share/gosplat/src/python_helper/fast_model_compare.py"
```
Comment (Gabriel-Ivarsson) [inline main.go:13]: Should be updated to the paths it was before after done

Comment (NoahHakansson) [reply main.go:13]: Will revert after PR is done and local testing is completed. :+1:

**Review by Gabriel-Ivarsson — CHANGES_REQUESTED** [review]
See comment. Otherwise very good code

**Review by Gabriel-Ivarsson — APPROVED** [review]
When changes reverted, good to go

**Pull request merged**